content/departments/cloud/technical-docs/v2.0/index.md

```diff
@@ -81,6 +81,81 @@ The following processes only apply to Cloud v2.0:
 - [Delete a Managed instance](./delete_process.md)
 - [Disaster Recovery process for a Managed instance](./disaster_recovery_process.md)
 
+### How to work with Cloud instances?
+
+> NOTE: This is the prereq for all things related to Cloud. Please review this section carefully. If you have questions, please reach out to #cloud and tag `@cloud-support`.
+
+Below is the bare minimal prereq before you can work with Cloud instances
+
+- [sourcegraph/cloud]: deployment repo where we persist all Cloud instances config and deployment artifacts
+- [sourcegraph/controller]: mi2 - cloud controller source code
+- Install `mi2` by following [sourcegraph/controller#installation](https://github.com/sourcegraph/controller#installation)
+- Sufficient access to GCP projects, see below FAQ to learn how to request access.
+
+Let's walkthrough the process of accessing a Cloud instance:
+
+First [locate the instance you are looking for](#how-to-locate-a-cloud-instance-in-the-deployment-repo)
+
+- `gh repo clone sourcegraph/cloud`
+- `cd environments/$ENVIRONMENT/deployments/$INSTANCE_ID`
+
+Then you can start running various `mi2` commands to work with a specific Cloud Instance (where we will infer the current instance base on current working directory).
+
+```sh
+# start a proxy to the database instance
+mi2 instance db proxy
+```
+
+Learn more from the `mi2` cli [reference](https://github.com/sourcegraph/controller/blob/main/reference.md) for detail usage and examples.
+
+### How to request access to Cloud instances infrastructure?
+
+We utilize Entitle to provide time-bound access to GCP infrastructure for both production and development environment.
+
+Use the slash command in Slack, type `/access_request` in any chat window and hit enter. Fill out the following values:
+
+- **Search permission**: One of `Cloud V2 Dev Access`, `Cloud V2 Prod Access`
+- **Permission duration**: Preferably to request the minimal amount of time
+- **Add justification**: Add a note to provide context why access is needed
```

Comment:
Does this apply to the cloud team as well or only other engineers?

Reply:
once Entitle is GA (soon), it will be applicable to all engineers (cloud or non-cloud)

however, cloud and non-cloud have different approval flow, cloud engineer will be auto-approved and non-cloud will require stamp from one of cloud, security, or their direct manager.